api-specificatie/ztc/openapi.yaml

```diff
@@ -2566,8 +2566,8 @@ components:
             ZAAKTYPE.
           type: string
           maxLength: 1000
-        isVan:
-          title: Is van
+        zaaktype:
+          title: Zaaktype
```

Comment:
Kan de title "Is van" worden? Net als bij Catalogus de title "Maakt deel uit van" is?